pkg/expect/expect_test.go

```diff
@@ -46,6 +57,8 @@ func TestExpectFunc(t *testing.T) {
 }
 
 func TestExpectFuncTimeout(t *testing.T) {
+	tailpath := getTailPath(t)
+
 	ep, err := NewExpect("tail", "-f", "/dev/null")
```

Comment:
It's not block comment: why not use `tailpath` as first argument of `NewExpect`?

Reply:
or, just use `tail` instead of full path